From cb14aea76b988402dd8558a1b1a3a40de915fdfa Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Sun, 18 Sep 2016 16:02:13 +0800
Subject: [PATCH] arm: configs: remove btsdio in linux defconfig

It will cause conflict if we are using other vendor driver.

Change-Id: I269c253fae874acf1b0290f5f16ca1433cf33b15
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index a7a45709f81b..416ea9af230b 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -127,7 +127,6 @@ CONFIG_BT=y
 CONFIG_BT_RFCOMM=y
 CONFIG_BT_HIDP=y
 CONFIG_BT_HCIBTUSB=y
-CONFIG_BT_HCIBTSDIO=y
 CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_ATH3K=y
 CONFIG_BT_HCIUART_LL=y
-- 
2.35.3

